scss/_list-group.scss

```diff
@@ -180,13 +180,25 @@
 // Add modifier classes to change text and background color on individual items.
 // Organizationally, this must come after the `:hover` states.
 
-@each $state, $value in $theme-colors {
-  $list-group-variant-bg: shift-color($value, $list-group-item-bg-scale);
-  $list-group-variant-color: shift-color($value, $list-group-item-color-scale);
-  @if (contrast-ratio($list-group-variant-bg, $list-group-variant-color) < $min-contrast-ratio) {
-    $list-group-variant-color: mix($value, color-contrast($list-group-variant-bg), abs($list-group-item-color-scale));
-  }
+@each $state in map-keys($theme-colors) {
```

Comment:
_Note: Still reviewing it so I don't know yet all the details of this PR_

I don't have an answer at the moment but something's very disturbing here.
If one wants to customize the colors one'd do something like:

```scss
$theme-colors: (
  "primary":    #7030a2,
  "secondary":  #0170ca,
  "success":    #fe0002,
  "info":       #f60,
  "warning":    #ffff01,
  "danger":     #00af52,
  "light":      $light,
  "dark":       $dark
) !default;
```

But here, in fact, it won't have any effects since we are mapping the names and not the values.
Same thing for alerts.
But the buttons would be updated with those new colors.

IDK if it can be done in v5.3 (probably not) but we should probably get rid of the values defined in `$theme-colors` OR maybe they should be defined with some CSS vars.

Anyway, this seems to be a breaking change here for projects using customized colors.